pysindy/pysindy.py

```diff
@@ -52,7 +52,20 @@ class SINDy(BaseEstimator):
         Feature library object used to specify candidate right-hand side features.
         This must be a class extending
         :class:`pysindy.feature_library.base.BaseFeatureLibrary`.
-        The default option is :class:`PolynomialLibrary`.
+        The default option is :class:`PolynomialLibrary`. If
+        inputs_per_library parameter is also passed, the library will use only
+        the subset of the inputs specified in this parameter. If
+        inputs_per_library is more than 1D, than feature_library must be a list
+        of feature library objects with the same dimension as
+        inputs_per_library.
+
+    inputs_per_library : np.ndarray, shape equal to
```

Comment:
Is there an example notebook we can point to which shows how this works?

Reply:
Nope, see the other comment -- this is defunct code. But I do show how it works with the Generalized Library in the Example 1 notebook